tests/phpunit/tests/test-wpml-config-locations.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+class WPML_Config_Locations_Test extends PLL_UnitTestCase {
+
+	protected static $dirs = array(
+		'themes' => array(
+			'best-theme',
+			'best-child',
+		),
+		'plugins' => array(
+			'best-plugin',
+		),
+	);
```

Comment:
There is no need of multidimensional arrays actually, this could simplify things a bit by removing one level of `foreach` in `wpSetUpBeforeClass()` and `wpTearDownAfterClass()`.
```php
	protected static $dirs = array(
		'themes/best-theme',
		'themes/best-child',
		'plugins/best-plugin',
	);
```

Reply:
Done in ea605a1b10c3ee74c22befe862613e7e1da76f5c.